crates/symbolicli/src/main.rs

```diff
@@ -0,0 +1,180 @@
+use std::io::Write;
+use std::{path::PathBuf, sync::Arc};
+
+use symbolicator_service::config::Config;
+use symbolicator_service::types::Scope;
+use symbolicator_sources::{SentrySourceConfig, SourceConfig, SourceId};
+
+use anyhow::{bail, Context};
+use clap::Parser;
+use reqwest::{header, Url};
+use serde::Deserialize;
+use tempfile::{NamedTempFile, TempPath};
+
+const BASE_URL: &str = "https://sentry.io/api/0/";
+
+#[tokio::main]
+#[allow(unreachable_code)]
+#[allow(unused)]
+async fn main() -> anyhow::Result<()> {
+    tracing_subscriber::fmt::init();
+
+    let args = Cli::parse();
+
+    let auth_token = std::env::var("SENTRY_AUTH_TOKEN").context("No auth token provided")?;
+
+    let config = Config::get(args.config.as_deref())?;
+
+    let runtime = tokio::runtime::Handle::current();
+    let (symbolication, _objects) =
+        symbolicator_service::services::create_service(&config, runtime)
+            .await
+            .context("failed to start symbolication service")?;
+
+    let org = args.org;
+    let project = args.project;
+    let event_id = args.event;
+
+    let mut headers = header::HeaderMap::new();
+    headers.insert(
+        header::AUTHORIZATION,
+        header::HeaderValue::from_str(&format!("Bearer {auth_token}")).unwrap(),
+    );
+
+    let client = reqwest::Client::builder()
+        .default_headers(headers)
+        .build()
+        .unwrap();
+
+    let minidump_path = download_minidump(&client, &org, &project, &event_id).await?;
+
+    tracing::info!(path = ?minidump_path, "minidump file downloaded");
+
+    let uploaded_difs = SourceConfig::Sentry(Arc::new(SentrySourceConfig {
+        id: SourceId::new("Uploaded DIFs"),
```

Comment:
```suggestion
        id: SourceId::new("sentry:project"),
```

Doesn’t really matter, but its what upstream uses IIRC.